prodtests/full-system-test/dpl-workflow.sh

```diff
@@ -6,6 +6,10 @@ command -v zsh > /dev/null 2>&1 && MYDIR=$(dirname $(zsh -c 'echo ${0:A}' "$0"))
 test -z ${MYDIR+x} && MYDIR="$(dirname $(readlink -f $0))"
 source $MYDIR/setenv.sh
 
+if [ -z $FILEWORKDIRRUN ]; then FILEWORKDIRRUN=$FILEWORKDIR; fi              # directory where to find the run-related files (grp, collision context)
+
+if [ ! -z $ISPILOTBEAMDATA ]; then BEAMTYPE=pp; fi  # pilot beam was pp
```

Comment:
I think the `BEAMTYPE` should come externally, and not be changed here.
BTW. which this check also `ISPILOTBEAMDATA=0` will lead to `BEAMTYPE=pp`

Reply:
Yes, we don't care about the value, provided it is there.
But ok, we will refactorize with an external script.